### ISSUE TITLE
Evaluation of SimpleRLPlayer not working

### ISSUE THREAD

**Issue by DivertisseurGames** [hsahovic/poke-env#349]
The training is going fine but when it tries to evaluate the player against a random one it just stops working.
I'm not getting any error, it just doesn't print anything.
Is it just because my pc is taking very long? idk.
Here's the full output : 
UserWarning: WARN: Official support for the `seed` function is dropped. Standard practice is to reset gym environments using `env.reset(seed=<desired seed>)`
  logger.warn(
C:\Users\Mael\AppData\Local\Programs\Python\Python310\lib\site-packages\gym\utils\env_checker.py:169: UserWarning: WARN: `return_info` is deprecated as an optional argument to `reset`. `reset`should now always return `obs, info` where `obs` is an observation, and `info` is a dictionarycontaining additional information.
  logger.warn(
C:\Users\Mael\AppData\Local\Programs\Python\Python310\lib\site-packages\gym\utils\passive_env_checker.py:233: DeprecationWarning: `np.bool8` is a deprecated alias for `np.bool_`.  (Deprecated NumPy 1.24)
  if not isinstance(terminated, (bool, np.bool8)):
2023-01-06 13:17:47.279881: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-01-06 13:17:47.312055: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:357] MLIR V1 optimization pass is not enabled
2023-01-06 13:17:47.423380: W tensorflow/c/c_api.cc:291] Operation '{name:'dense_2_1/kernel/Assign' id:179 op device:{requested: '', assigned: ''} def:{{{node dense_2_1/kernel/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_FLOAT, validate_shape=false](dense_2_1/kernel, dense_2_1/kernel/Initializer/stateless_random_uniform)}}' was changed by setting attribute after it was run by a session. This mutation 
will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
WARNING:tensorflow:From C:\Users\Mael\AppData\Local\Programs\Python\Python310\lib\site-packages\tensorflow\python\autograph\pyct\static_analysis\liveness.py:83: Analyzer.lamba_check (from tensorflow.python.autograph.pyct.static_analysis.liveness) is deprecated and will be removed after 2023-09-23.
Instructions for updating:
Lambda fuctions will be no more assumed to be used in the statement where they are used, or at least in the same block. https://github.com/tensorflow/tensorflow/issues/56089
Training for 10000 steps ...
Interval 1 (0 steps performed)
C:\Users\Mael\AppData\Local\Programs\Python\Python310\lib\site-packages\keras\engine\training_v1.py:2357: UserWarning: `Model.state_updates` will be removed in a future version. This property should not be used in TensorFlow 2.0, as `updates` are applied automatically.
  updates=self.state_updates,
2023-01-06 13:17:49.424296: W tensorflow/c/c_api.cc:291] Operation '{name:'dense_2/BiasAdd' id:95 op device:{requested: '', assigned: ''} def:{{{node dense_2/BiasAdd}} = BiasAdd[T=DT_FLOAT, _has_manual_control_dependencies=true, data_format="NHWC"](dense_2/MatMul, dense_2/BiasAdd/ReadVariableOp)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2023-01-06 13:17:49.455286: W tensorflow/c/c_api.cc:291] Operation '{name:'count_2/Assign' id:305 op device:{requested: '', assigned: ''} def:{{{node count_2/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_FLOAT, validate_shape=false](count_2, count_2/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
 1001/10000 [==>...........................] - ETA: 2:30 - reward: 0.35712023-01-06 13:18:06.349830: W tensorflow/c/c_api.cc:291] Operation '{name:'dense_2_1/BiasAdd' id:189 op device:{requested: '', assigned: ''} def:{{{node dense_2_1/BiasAdd}} = BiasAdd[T=DT_FLOAT, _has_manual_control_dependencies=true, data_format="NHWC"](dense_2_1/MatMul, dense_2_1/BiasAdd/ReadVariableOp)}}' was changed by setting attribute after it was run by 
a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2023-01-06 13:18:06.634700: W tensorflow/c/c_api.cc:291] Operation '{name:'loss_3/AddN' id:411 op device:{requested: '', assigned: ''} def:{{{node loss_3/AddN}} = AddN[N=2, T=DT_FLOAT, _has_manual_control_dependencies=true](loss_3/mul, loss_3/mul_1)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2023-01-06 13:18:06.685677: W tensorflow/c/c_api.cc:291] Operation '{name:'training/Adam/dense_2/kernel/v/Assign' id:671 op device:{requested: '', assigned: ''} def:{{{node training/Adam/dense_2/kernel/v/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_FLOAT, validate_shape=false](training/Adam/dense_2/kernel/v, training/Adam/dense_2/kernel/v/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
10000/10000 [==============================] - 297s 30ms/step - reward: 0.7144
done, took 298.200 seconds

and then nothing after the "done"

**Comment by KlausRyu** [comment]
hey i got same error how do you fix it

**Comment by cameronangliss** [comment]
This example code no longer exists as it has been replaced, so I will close this.
